docs/collator/SetupAndRun/configuration.md

```diff
@@ -0,0 +1,340 @@
+---
+sidebar_position: 2
+sidebar_label: 📝 Configuration
+title: 🚄 Setup and run a Collator
+hide_title: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[Installation](installation) > Configuration > [Running](running) > [Session keys](keys) > [Sync](sync) > [Wait](wait)
+
+## 📝 Configuration
+some extra command line parameters are required or helpful for collating.
+
+<Tabs groupId="os">
+<TabItem value="fedora" label="fedora">
+
+edit the calamari service unit file to include collation parameters in the `ExecStart` command.
+
+`/usr/lib/systemd/system/calamari.service`
+
+```ini
+ExecStart=/usr/bin/calamari \
+    --collator \
+    --name 'my parachain collator node name' \
+    --chain /usr/share/substrate/calamari.json \
+    --base-path /var/lib/substrate \
+    --port 31333 \
+    --ws-port 9144 \
+    --ws-max-connections 100 \
+    --rpc-port 9133 \
+    --rpc-cors all \
+    --rpc-methods auto \
+    --prometheus-port 9615 \
+    --prometheus-external \
+    --state-cache-size 0 \
+    --bootnodes \
+        /dns/crispy.calamari.systems/tcp/30333/p2p/12D3KooWNE4LBfkYB2B7D4r9vL54YMMGsfAsXdkhWfBw8VHJSEQc \
+        /dns/crunchy.calamari.systems/tcp/30333/p2p/12D3KooWL3ELxcoMGA6han3wPQoym5DKbYHqkWkCuqyjaCXpyJTt \
+        /dns/hotdog.calamari.systems/tcp/30333/p2p/12D3KooWBdto53HnArmLdtf2RXzNWti7hD5mML7DWGZPD8q4cywv \
+        /dns/tasty.calamari.systems/tcp/30333/p2p/12D3KooWGs2hfnRQ3Y2eAoUyWKUL3g7Jmcsf8FpyhVYeNpXeBMSu \
+        /dns/tender.calamari.systems/tcp/30333/p2p/12D3KooWNXZeUSEKRPsp1yiDH99qSVawQSWHqG4umPjgHsn1joci \
+    -- \
+    --name 'my embedded relay node name' \
+    --chain /usr/share/substrate/kusama.json \
+    --port 31334 \
+    --ws-port 9145 \
+    --rpc-port 9134 \
+    --prometheus-port 9616 \
+    --prometheus-external \
+    --telemetry-url 'wss://api.telemetry.manta.systems/submit/ 0'
+```
+
+</TabItem>
+<TabItem value="ubuntu" label="ubuntu">
+
+edit the calamari service unit file to include collation parameters in the `ExecStart` command.
+
+`/usr/lib/systemd/system/calamari.service`
+
+```ini
+ExecStart=/usr/bin/calamari \
+    --collator \
+    --name 'my parachain collator node name' \
+    --chain /usr/share/substrate/calamari.json \
+    --base-path /var/lib/substrate \
+    --port 31333 \
+    --ws-port 9144 \
```

Comment:
Seems we're not going to use all default ports, right?

Reply:
i have some thoughts on that, however this pr makes no changes to port numbers. this pr is only about the 3 points mentioned above (md rendering in tabs, split SetupAndRun, enable redirects).
lets discuss other changes in a future pr where we actually make those changes without changing the scope of this pr.